app/sprinkles/account/assets/userfrosting/js/uf-password-security.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  //setup before functions
+  var passwordInput = $("form").find('input[name=password]');
+  var typingTimer; //timer identifier
+  var doneTypingInterval = 200; //time in ms
+  var breachThreshold = site.password_security.enforce_no_compromised.breaches;
+
+
+  //on keyup, start the countdown
+  passwordInput.keyup(function() {
+    clearTimeout(typingTimer);
+    if (passwordInput.val()) {
+      typingTimer = setTimeout(doneTyping, doneTypingInterval);
+    }
+  });
+
+
+  jQuery.validator.addMethod("check-password", function(value, element, breaches) {
+    return breaches < breachThreshold;
+  }, jQuery.validator.format("The password security policy does not allow use of known compromised passwords. Please enter a different password."));
+
+  if (breachThreshold != -1) {
+    //user is "finished typing," do something
+    function doneTyping() {
```

Comment:
I can't see any reason for this to be wrapped in a conditional. Currently I can see this cause JS errors under certain scenarios as well.

Reply:
If this feature is disabled by setting `site.password_security.enforce_no_compromised.breaches` we do not want to run the client-side validation to run.

Reply:
My JS skills are not great. If there is another way to handle this I am happy to change it. Otherwise, I do not know of any other way to handle this.